aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -367,6 +368,11 @@ void gemm<at::BFloat16>(CUDABLAS_GEMM_ARGTYPES(at::BFloat16)) {
     // See Note [Writing Nondeterministic Operations]
     globalContext().alertCuBLASConfigNotDeterministic();
     cublasHandle_t handle = at::cuda::getCurrentCUDABlasHandle();
+#if defined(CUDA_VERSION) && CUDA_VERSION >= 11000
```

Comment:
Interesting that this check is redundant with the body of the guard struct. Should the guard struct itself be defined if this isn't true?

Reply:
I removed the check here, and makes `NoTF32Guard` a noop on CUDA < 11